lib/videoroom_web/templates/room/index.html.heex

```diff
@@ -14,15 +14,15 @@
             <template id="screensharing-template">
                 <div name="video-feed" class="relative bg-gray-900 shadow rounded-md overflow-hidden h-full w-full ratio-video">
                     <video class="w-full h-full"></video>
-                    <div name="video-label" class="absolute text-white text-shadow-lg bottom-0 left-0 p-2">Placeholder</div>
+                    <div name="video-label" class="absolute text-white text-shadow-lg bottom-0 left-0 p-2 z-50">Placeholder</div>
```

Comment:
I know it's a tiny fix, but it's still an independent fix, so unless there's a good reason such fixes should not be a part of one PR. Please extract that change and open a new PR

Reply:
Extracted to [#46](https://github.com/membraneframework/membrane_videoroom/pull/46)